test/parallel/test-http-status-code.js

```diff
@@ -55,13 +54,14 @@ function nextTest() {
     response.on('end', function() {
       testsComplete++;
       testIdx += 1;
-      nextTest();
+      if (countdown.dec())
+        nextTest();
     });
     response.resume();
   });
 }
 
 
 process.on('exit', function() {
-  assert.strictEqual(5, testsComplete);
+  assert.strictEqual(6, testsComplete);
```

Comment:
How about replacing `6` with `tests.length` ?

Reply:
This code can actually be removed, as well as the `testsComplete` variable. Since there's a Countdown now, there's no need to verify that all 6 test cases executed.